robolectric/src/test/java/org/robolectric/shadows/ShadowParcelFileDescriptorTest.java

```diff
@@ -117,11 +125,13 @@ public void testAdoptFd_canReadReadOnlyFile() throws Exception {
     assertThat(newPfd).isNotNull();
     assertThat(newPfd.getFileDescriptor().valid()).isTrue();
     is = new FileInputStream(newPfd.getFileDescriptor());
-    assertThat(is.read()).isEqualTo(READ_ONLY_FILE_CONTENTS_BYTE2);
+    assertThat(is.read()).isEqualTo(READ_ONLY_FILE_CONTENTS_THE_SECOND_BYTE);
   }
 
   @Test
   public void testAdoptFd_canWriteWritableFile() throws Exception {
+    assumeThat("Not supposed to be run on Windows.", File.separator, Matchers.equalTo("/"));
```

Comment:
What about extracting "Not supported to be run on Windows" to a constant string or extracting this `assumeThat` to a single method?

Reply:
Done.